iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -252,15 +249,31 @@ export class iXBRLViewer {
                 /* Poll for iframe load completing - there doesn't seem to be a reliable event that we can use */
                 let timer = setInterval(function () {
                     let complete = true;
-                    iframes.each(function (n) {
-                        const iframe = this;
+                    iframes.each((n, iframe) => {
                         const iframeDoc = iframe.contentDocument || iframe.contentWindow.document;
                         if ((iframeDoc.readyState != 'complete' && iframeDoc.readyState != 'interactive') || $(iframe).contents().find("body").children().length == 0) {
                             complete = false;
                         }
                     });
                     if (complete) {
                         clearInterval(timer);
+                        const html = $('html');
+                        const lang = html.attr('lang') || html.attr('xml:lang');
+                        // Remove any report language from the application html node.
+                        html.removeAttr('lang');
+                        html.removeAttr('xml:lang');
+                        iframes.each((n, iframe) => {
+                            const iframeDoc = iframe.contentDocument || iframe.contentWindow.document;
+                            const iframeHtml = $(iframeDoc).find('html');
+                            let frameLang = iframeHtml.attr('lang') || iframeHtml.attr('xml:lang');
+                            if (frameLang === undefined) {
+                                frameLang = lang;
+                            }
+                            if (frameLang !== undefined) {
+                                // Try to set the language of the iframe to the language of the report.
```

Comment:
I believe this explains what we're trying to do?
> For accessibility, the lang attribute should be set, but XHTML does not allow that, so copy the xml:lang attribute into the lang attribute.

The only detail I'm unsure on that would be nice to have in the comment is whether this is actually required for accessibility tools when the document is served as XHTML (vs HTML).  Perhaps the created iframe can never be served as XHTML?  Or perhaps the dynamically created iframe inherits the XHTML-ness of the parent document?

Anyway, it seems like we can simplify to just this?  Otherwise, I don't understand why `.attr('lang')` is necessary (since it's not valid XHTML) or why we would copy into `xml:lang` (what software would care?).
```
const frameLang = iframeHtml.attr('xml:lang');
if (frameLang !== undefined) {
  iframeHtml.attr('lang', frameLang);
}
```

Reply:
> We currently set the lang attribute to a default of en-US and only for non stub viewers. This change removes the default and migrates the lang attribute to report iframes so as not to contradict the UI language.

Is this definitely the logic we want?  I think it's still only non-stub mode that we care about, in which case we should migrate the parent `xml:lang` attribute, if any, to the first (re-parented) iframe.

For stub viewer mode, and additional non-stub iframes, I think we should leave them untouched.

I don't think we need to check for `lang` attributes because they'd make the document invalid if present (no harm in deleting them just to make sure, though).

Reply:
Simple enough. Updated.